typings/objection/index.d.ts

```diff
@@ -471,6 +471,7 @@ declare namespace Objection {
     // Non-query methods:
 
     context(queryContext: object): this;
+    context(): object;
```

Comment:
`object` ans an argument is ok since any object can be downcasted to `object`. The argument shouldn't be `QueryContext` because `QueryContext` type contains some properties that are added by objection and should never be passed to `context` or `mergeContext` methods.